kitsune/sumo/static/sumo/js/instant_search.js

```diff
@@ -10,13 +10,22 @@
     $('#main-content').hide();
     $('#main-content').siblings('aside').hide();
     $('#main-breadcrumbs').hide();
+
+    if ($('#support-search-wiki:visible').length === 0) {
+      $('.support-search-main').show();
+      $('.support-search-main').find('input[name=q]').focus();
+    }
   }
 
   function showContent() {
+    $('.support-search-main').hide();
     $('#main-content').show();
     $('#main-content').siblings('aside').show();
     $('#main-breadcrumbs').show();
     $('#instant-search-content').remove();
+    $('.search-form-large:visible').find('input[name=q]').focus();
+    $('.search-form-large:visible').find('input[name=q]').val('');
```

Comment:
Nit - I think we can chain these calls? E.g. `$('.search-form-large:visible').find('input[name=q]').focus().val('');`

Reply:
Ah you're right @jpetto ! Chained them now thanks!